src/Uno.UI.Composition/Composition/CompositionSpriteShape.skia.cs

```diff
@@ -141,9 +131,63 @@ private static SKPaint TryCreateAndClearPaint(in Visual.PaintingSession session,
 				}
 			}
 
-			paint.ColorFilter = session.Filters.OpacityColorFilter;
```

Comment:
I get why you want to remove it ... but it has the advantage to make sure that we don t forget to do it. If there is no other reason than code clarity, I think I prefer the previous version.

Reply:
I moved it out of the function because I need to use the function (TryCreateAndClearPaint) in the hit-testing logic (that runs without a `session`). I guess I can create an additional function that redirects to this one + adds this line back so we can keep both versions.

Reply:
I adjust the signature to require a (possibly null) `colorFilter` parameter.